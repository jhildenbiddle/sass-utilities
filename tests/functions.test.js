const compileSass = require('./helpers/compile-sass');

describe('chevron-data-uri', () => {
    test('Compile', () => {
        const result = compileSass(`
            .myclass {
                background: red;

                nested {
                    color: blue;
                }
            }
        `, false, false);

        expect(result).toMatchSnapshot();
    });
});
