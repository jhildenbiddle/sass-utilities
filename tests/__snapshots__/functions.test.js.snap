// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`chevron-data-uri Compile 1`] = `
".myclass {
  background: red;
}
.myclass nested {
  color: blue;
}"
`;
